src/redux/reducers.js

```diff
@@ -1020,11 +1020,14 @@ export function users(state = {}, action) {
       return mergeByIds(state, (action.payload.users || []).map(userParser))
     }
     case ActionTypes.REALTIME_POST_NEW:
+    case ActionTypes.REALTIME_LIKE_NEW:
     case ActionTypes.REALTIME_COMMENT_NEW: {
-      return mergeByIds(state, (action.users || []).map(userParser))
-    }
-    case ActionTypes.REALTIME_LIKE_NEW: {
-      return mergeByIds(state, ([action.user]).map(userParser))
+      const safeUsers = action.users || [{}]
+      const userAlreadyAdded = state[safeUsers[0].id]
+      if (userAlreadyAdded) {
+        return state
+      }
+      return mergeByIds(state, safeUsers.map(userParser))
```

Comment:
It's almost ok, cause no `id` here means no matches on component state composition. Could change to guardian if, but don't know whether this'll be that much better.